src/averell/readers/goldpoems.py

```diff
@@ -3,18 +3,22 @@
 
 def parse_xml(xml_file):
     """
-    XML TEI poem parser for ADSO 100 poems corpus
-    :param xml_file: path of xml file
-    :return: poem dict
+    XML TEI poem parser for 'ADSO 100 poems' corpus.
+    We read the data and find elements like title, author, etc with XPath
+    expressions.
+    Then, we iterate over the poem text and we look for each stanza and line
+    data.
+    :param xml_file: Path for the xml file
+    :return: Poem python dict with the data obtained
     """
+    ns = "{http://www.tei-c.org/ns/1.0}"
```

Comment:
I think we should add this as a constant

Reply:
TEI_NAMESPACE = "{http://www.tei-c.org/ns/1.0}" added in utils.py